### PR TITLE
tests: bluetooth: tester: add config files for nrf54l15dk

### PR DESCRIPTION
**Pull request by m-alperen-sener**
- Add condig files for nrf54l15dk.
- Move board config files under boards directory.
- Remove target config files that are not used, obsolete.
- Move target specific kconfig to board config files.

### CODE REVIEW

File: tests/bluetooth/tester/prj.conf [hunk 1]
```diff
@@ -86,7 +86,6 @@ CONFIG_SETTINGS=y
 CONFIG_FLASH=y
 CONFIG_FLASH_MAP=y
 CONFIG_NVS=y
```
[the remark this answers is not in the document]
Comment (m-alperen-sener) [reply tests/bluetooth/tester/prj.conf:88]: IMO that should be another PR or commit; we can have it after https://github.com/nrfconnect/sdk-nrf/pull/17502

[the remark this answers is not in the document]
Comment (m-alperen-sener) [reply tests/bluetooth/tester/prj.conf:88]: Yeap I understand it now, actually. yes we can do that.

File: tests/bluetooth/tester/prj.conf [hunk 2]
```diff
@@ -81,16 +113,3 @@ CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_FADE_PROLONG=0
...
 CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_PROLONG=0
-
-CONFIG_SETTINGS=y
-CONFIG_FLASH=y
-CONFIG_FLASH_MAP=y
-CONFIG_NVS=y
-CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE=y
-
-CONFIG_BT_TINYCRYPT_ECC=y
-CONFIG_BT_TESTING=y
```
Comment (Balaklaka) [inline tests/bluetooth/tester/prj.conf:92]: BT_TESTING has been removed?

Comment (m-alperen-sener) [reply tests/bluetooth/tester/prj.conf:92]: https://github.com/nrfconnect/sdk-zephyr/blob/main/subsys/bluetooth/host/Kconfig#L1032

Comment (m-alperen-sener) [reply tests/bluetooth/tester/prj.conf:92]: It is better to add back even though it is not needed for mmdl test cases, just to keep aligned with upstream tester.

**Review by Balaklaka — APPROVED** [review]

File: tests/bluetooth/tester/boards/nrf54l15dk_nrf54l15_cpuapp.conf [hunk 3]
```diff
@@ -0,0 +1 @@
+CONFIG_NVS=y
```
Comment (PavelVPV) [inline tests/bluetooth/tester/boards/nrf54l15dk_nrf54l15_cpuapp.conf:1]: Not ZMS? It is merged into ncs now.

Comment (m-alperen-sener) [reply tests/bluetooth/tester/boards/nrf54l15dk_nrf54l15_cpuapp.conf:1]: actually we can do this now. 👍

**Review by ludvigsj — APPROVED** [review]

**Review by PavelVPV — APPROVED** [review]

**Pull request merged**